### PR TITLE
Simplify expression and tweak annotation.

### PR DESCRIPTION
**Pull request by ryanve**
This gives the same result. Based on the [annotation](https://developers.google.com/closure/compiler/docs/js-for-compiler) docs I think that `Boolean` implies `instanceof Boolean`  whereas `boolean` implies the `typeof`.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /**
  * Check if offline.
  *
- * @return {Boolean} offline
+ * @return {boolean} offline
```
Comment (juliangruber) [inline index.js:4]: This describes what type is returned, not how you get that type.

**Comment by juliangruber** [comment]
can you rebase and squash?

File: index.js (hunk 1 above)
Comment (juliangruber) [inline index.js:4]: nvm I see what you're saying now. Can you still squash please?

**Comment by ryanve** [comment]
Yea, the simplification and the annotation were independent changes. It should be ready now.

**Comment by juliangruber** [comment]
cool, thanks!

**Comment by ryanve** [comment]
Rock on =)

**Pull request merged**